modules/shared/components/Select.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Select } from "@chakra-ui/react";
+
+interface Category {
+  option: string[];
+  label: string;
+}
+
+const ComponentSelect = ({ label, option }: Category) => {
+  return (
+    <>
+      <label htmlFor={label}>{label}</label>
+      <Select placeholder="Select option">
```

Comment:
Y utilizar esas props, para que podamos tambien pasarle las props de `value`, `onChange` etc..

```suggestion
const ComponentSelect = ({ label, option, ...props }: Props) => {
  return (
    <>
      <label htmlFor={label}>{label}</label>
      <Select {...props} placeholder="Select option">
```